Update whats new 3 47 (#13722)
flutter/website: sites/docs/src/content/release/whats-new.md

[flutter-announce]: {{site.groups.flutter-announce}}
[release notes]: /release/release-notes

## 12 August 2026: Release 3.47

Flutter 3.47 is live! For details,
check out the [Flutter 3.47 blog post][3.47-blog-post].

**Docs updated or added since the 3.44 release**

* The Flutter [Widget Previews][widget-previewer] tool has graduated to stable,
  introducing local build caching for faster startup times
  and real-time preview search and filtering.

* The AI documentation has been reorganized to improve navigation
  and help you get started quickly with agent-plugins
  (formerly known as skills) using the [Antigravity CLI][antigravity-cli],
  IDE integrations, and other coding assistants.
  An experimental [AI Evaluations][ai-evals] topic page has also been added.

* There is a new guide page on [Dependency management][dependency-mgmt]

>   to help you configure, manage, and upgrade package dependencies.

Reviewer: ```suggestion
* A new guide on [Dependency management][dependency-mgmt] explains
  how to configure, manage, and upgrade package dependencies.
```

Author: Added.

* The [Swift Package Manager][swiftpm] guide for iOS and macOS has been
  expanded, and the [Add-to-App][add-to-app] documentation has been updated

>   with SwiftPM integration guides.

Reviewer: ```suggestion
* The [Swift Package Manager][swiftpm] guides for iOS and macOS have expanded,
  and the [Add-to-app][add-to-app] documentation now includes
  SwiftPM integration instructions.
```

Author: Added.

* The web documentation now includes troubleshooting guidance for
  [Wasm compilation errors][wasm] and instructions for using the
  `--source-maps` flag during web builds.

* A new section has been added to the assets guide explaining how to bundle

>   [platform-specific assets][platform-assets] in `pubspec.yaml`.

Reviewer: ```suggestion
* The assets guide now explains how to bundle
  [platform-specific assets][platform-assets] in `pubspec.yaml`.
```

Author: Added.

* Guidance has been added for migrating plugins and apps to use

>   [built-in Kotlin][kotlin-migration] Gradle plugins.

Reviewer: ```suggestion
* A new migration guide helps you migrate plugins and apps to
  [built-in Kotlin][kotlin-migration] Gradle plugins.
```

Author: Added.
